### ISSUE TITLE
Renaming of database package failes

### ISSUE THREAD

**Issue by btwe** [irods/irods#2493]
As requested from #1301 I open a new issue.

First the metadata:
OS: debian 7.8 (minimal installation)
iRODS: 4.0.3-696-g31c9a08

After installing dependencies `build.sh` fails after compiling, when it tries to rename the database package:
```bash
irods@ir-z00-ies:~/local/src/irods$ ./packaging/build.sh -p icat postgres

+------------------------------------+
| RENCI iRODS Build Script           |
+------------------------------------+
Mon Jan 26 09:05:52 CET 2015

-p detected -- Building portable package

Detecting Build Environment
Detected Packaging Directory [/home/irods/local/src/irods/packaging]
Build Directory set to [/home/irods/local/src/irods]
Detected iRODS Version to Build [4.1.0]
Detected iRODS Version Integer [401000]
Detected OS [Portable]
Detected OS Version [7.8]
Detected Python.h [/usr/include/python2.7/Python.h]
Detected curl [/usr/bin/curl] v[7.26.0]
Detected wget [/usr/bin/wget] v[1.13.4]
Detected help2man [/usr/bin/help2man] v[1.40.10]
Detected unixODBC library [     libodbc.so.1 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libodbc.so.1
        libodbc.so (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libodbc.so]
Detected libfuse library [/usr/include/fuse.h
/usr/include/fuse/fuse.h]
Detected libcurl library [/usr/include/curl/curl.h]
Detected bzip2 library [/usr/include/bzlib.h]
Detected zlib library [/usr/include/zlib.h]
Detected pam library [/usr/include/security/pam_appl.h]
Detected OpenSSL sha.h library [/usr/include/openssl/sha.h]
Detected libxml2 library [/usr/include/libxml2/libxml/parser.h]
Detected krb5 library [/usr/include/gssapi.h
/usr/include/gssapi/gssapi.h
/usr/include/mit-krb5/gssapi.h
/usr/include/mit-krb5/gssapi/gssapi.h]
-------------------------------------
Detected CPUs:    2
Compiling with:   make -j 5
-------------------------------------
-----------------------------
Configuring and Building iRODS
-----------------------------
Configure iRODS
------------------------------------------------------------------------

Step 1 of 3:  Verifying configuration...
    No database configured.

Step 2 of 3:  Checking host system...
    Host OS is Linux.
    Perl:        /usr/bin/perl
    C compiler:  /usr/bin/g++ (gcc)
      Flags:     none
    Loader:      /usr/bin/g++
      Flags:     none
    Archiver:    /usr/bin/ar
    Ranlib:      /usr/bin/ranlib

Step 3 of 3:  Updating configuration files...
    Updating config.mk...
        Created /home/irods/local/src/irods/iRODS/config/config.mk
    Updating platform.mk...
        Created /home/irods/local/src/irods/iRODS/config/platform.mk
    Updating irodsctl...

Done.
make: Entering directory `/home/irods/local/src/irods'

[...]

Building Database Plugins
g++ -Wno-write-strings -DRODS_SERVER -I../../../iRODS/lib/core/include/ -I../../../iRODS/lib/api/include/ -I../../../iRODS/lib/md5/include -I../../../iRODS/server/core/include/ -I../../../iRODS/server/icat/include/ -I../../../iRODS/server/re/include/ -I../../../iRODS/server/drivers/include/ -I/home/irods/local/src/irods/external/libarchive-3.1.2/libarchive/ -I/home/irods/local/src/irods/external/boost_1_57_0 -I/usr/include/openssl -I../../../iRODS/lib/sha1/include -I../../../iRODS/../plugins/database/include -I../../../external/jansson-2.7/src/ -Dlinux_platform  -ldl -rdynamic -fPIC -ldl -rdynamic -lm -lpthread -lcurl -lssl -lcrypto "-Wl,-E"  -shared -o .././libmysql.so  .objs/db_plugin.o  .objs/general_query.o  .objs/general_update.o  .objs/general_query_setup.o  .objs/mid_level_routines.o  .objs/low_level_odbc.o  .objs/irods_catalog_properties.o  .objs/irods_sql_logger.o ../../../iRODS/lib/core/obj/irods_plugin_base.o -lodbc -L/lib -lnsl -lm
make[1]: Leaving directory `/home/irods/local/src/irods/plugins/database/mysql'
Running EPM :: Generating Debian DEBs
epm: Product names should only contain letters and numbers!
epm: Warning - file permissions and ownership may not be correct
     in Debian packages unless you run EPM as root!
Building development package archive file...
Running EPM :: Generating Portable TGZs
epm: Product names should only contain letters and numbers!
epm: Product names should only contain letters and numbers!
epm: Product names should only contain letters and numbers!
EPM produced packages:
./linux-3.2-x86_64/irods-dev-4.1.0-linux-3.2-x86_64.tar.gz
./linux-3.2-x86_64/irods-icat-4.1.0-linux-3.2-x86_64.tar.gz
./linux-3.2-x86_64/irods-runtime-4.1.0-linux-3.2-x86_64.tar.gz
ls: cannot access ./plugins/database/*/*database*.tar.gz: No such file or directory


renaming    [./[lm]*/irods-dev-*4.1.0*.tar.gz]
         to [./build/irods-dev-4.1.0-64bit.tar.gz]

renaming    [./[lm]*/irods-runtime-*4.1.0*.tar.gz]
         to [./build/irods-runtime-4.1.0-64bit.tar.gz]

renaming    [./[lm]*/irods-*4.1.0*.tar.gz]
         to [./build/irods-icat-4.1.0-64bit.tar.gz]

renaming    [./plugins/database/*/*database*.tar.gz]
         to [./build/*database*.tar.gz]
mv: cannot stat `./plugins/database/*/*database*.tar.gz': No such file or directory
```

Same (or similar) failure happens when doing the build with `mysql` or `postgres` or with or without `-p` switch.

**Comment by trel** [comment]
I just tried on Debian 7 and completed a build, installation, and test without any trouble.

I did not use the -p option (we'll probably be removing it soon).

Can you try to clean your repository and build again?

```
cd irods
git clean -xffd
./packaging/build.sh icat postgres
```

**Comment by btwe** [comment]
@trel Many thanks, Now the build process finishes without error.

Some details: Two things changed in comparison to my last runs:

1. The file `irods_external.tar.gz` which is fetched from `ftp://ftp.renci.org/pub/irods/build/irods_external_Debian.tar.gz` has been updated in the meantime. Its timestamp is now from jan 23 2015 and it contains the avrocpp part which was missing before. I had to build the external tarball my self via `cd external; make clean ; make generate`. I did not trace any differences in there, sorry.

2. Before running the build process again I did a `git fetch origin && git merge origin/master`.
